tree/ntuple/v7/test/ntuple_storage_daos.cxx

```diff
@@ -217,6 +217,34 @@ TEST_F(RPageStorageDaos, MultipleNTuplesPerContainer)
    EXPECT_THROW(RNTupleReader::Open("ntuple3", daosUri), ROOT::Experimental::RException);
 }
 
+TEST_F(RPageStorageDaos, DisabledSamePageMerging)
+{
+   std::string daosUri = RegisterLabel("ntuple-test-disabled-same-page-merging");
+   auto model = RNTupleModel::Create();
+   model->MakeField<float>("px", 1.0);
+   model->MakeField<float>("py", 1.0);
+   RNTupleWriteOptionsDaos options;
```

Comment:
What happens if I call `options.SetEnablePageChecksums(true)` here? Should this be covered by a test case?

Reply:
Checksums are anyway turned on by default, but I think it makes sense to set it here explicitly. Fixed.